app/models/compute_resources/foreman/model/gce.rb

```diff
@@ -119,7 +143,7 @@ def check_google_key_path
 
     def vm_instance_defaults
       super.merge(
-        :zone_name => zone,
+        :zone      => zone,
```

Comment:
can we stop using these horrible indented items? every change causes horrible merge issues just because it "looks better on the eye" (it doesn't, IMHO)
should be
```ruby
:zone => zone,
:long_name_for_example => "whatever"
```